src/workflows-app/SubmissionDetails.js

```diff
@@ -400,6 +406,20 @@ export const BaseSubmissionDetails = ({ name, namespace, workspace, submissionId
                             return h(TextCell, [customFormatDuration(paginatedPreviousRuns[rowIndex].duration)]);
                           },
                         },
+                        {
+                          size: { basis: 400, grow: 0 },
+                          field: 'workflowId',
+                          headerRenderer: () => h(Sortable, { sort, field: 'workflowId', onSort: setSort }, ['Workflow ID']),
+                          cellRenderer: ({ rowIndex }) => {
+                            const engineId = paginatedPreviousRuns[rowIndex].engine_id;
+                            if (engineId !== undefined) {
```

Comment:
I think we'll still want to have something in the cell even if the engine id isn't defined, otherwise this might still cause an error 🤔 Not sure what makes sense to show to users

Reply:
It worked okay when I tried, just displayed an empty cell.